### ISSUE TITLE
I can model the message marker of a message flow

### ISSUE THREAD

**Issue by gimbel** [bpmn-io/bpmn-js#447]
note: if the marker is visible it should be possible to give it a label too

**Comment by nikku** [comment]
See [forum topic](https://forum.bpmn.io/t/add-message-symbol/705), too.

**Comment by kduvekot** [comment]
Its an old request ... but is there anything I can do to help this along?

**Comment by nikku** [comment]
Yes. You can contribute a pull request that implements this. :rocket:
